xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1497,7 +1497,7 @@ void CGUIWindowVideoBase::LoadPlayList(const std::string& strPlayList, int iPlay
     // load it
     if (!pPlayList->Load(strPlayList))
     {
-      CGUIDialogOK::ShowAndGetInput(6, 0, 477, 0);
+      CGUIDialogOK::ShowAndGetInput(6, 477);
```

Comment:
That ID 6 is used many, many times. Best first find out where it's used and then change that in one go?